Dockerfile.template

```
@@ -137,7 +137,7 @@ RUN set -ex; \
 
 # gcc installs .so files in /usr/local/lib64...
 RUN set -ex; \
-	echo '/usr/local/lib64' > /etc/ld.so.conf.d/local-lib64.conf; \
+	sed -i '1s/^/\/usr\/local\/lib64\n/' /etc/ld.so.conf; \
```

Whoa, great catch -- however, I don't think we need to get quite this complicated.  It should be enough for us to rename `local-lib64.conf` to something that sorts above `aarch64` (and any other architecture string), so this should be enough:

```suggestion
	echo '/usr/local/lib64' > /etc/ld.so.conf.d/000-local-lib64.conf; \
```

(This also made me realize this is probably broken completely on 32bit architectures like `arm32v7`, but I think that's something we should solve separately. :smile:)